Cargo.toml

```diff
@@ -24,7 +24,7 @@ libc = { version = "0.2.58", optional = true }
 xenctrl = { git = "https://github.com/Wenzel/xenctrl", optional = true }
 xenstore = { git = "https://github.com/Wenzel/xenstore", optional = true }
 xenforeignmemory = { git = "https://github.com/Wenzel/xenforeignmemory", optional = true }
-kvmi = { git = "https://github.com/Wenzel/kvmi", optional = true }
+kvmi = { git = "https://github.com/arnabcs17b006/kvmi", optional = true }
```

Comment:
Do you have unmerged changes in your own kvmi fork?

Reply:
Yes. I made ffi_event a public member of event struct.

Reply:
Ah, I see. Could you create a PR for that?

Reply:
Sure👍